src/polyfills/details/_base.scss

```diff
@@ -6,6 +6,8 @@
 summary {
 	// Make sure summary remains visible
 	display: list-item !important;
+	list-style-type: none; // Prevent list bullets from appearing in browsers that lack native details/summary support (e.g. IE/Edge)
+	list-style-type: disclosure-closed; // Show summary arrows in Firefox 49.0+ and future versions of Blink/Webkit
```

Comment:
@duboisp Since I'm no longer using ``list-style`` (shorthand for ``list-style-type``, ``list-style-position`` and ``list-style-image``) and am now solely using ``list-style-type``, the ``inside`` value isn't needed anymore.

I guess this works out for the best since things are now more-targeted than before and are no longer re-declaring as many of Firefox's default ``summary`` properties.